src/components/MoviesList.js

```diff
@@ -1,27 +1,13 @@
 import React from 'react';
 import MovieTile from './MovieTile.js';
 
-export default function MoviesList (props) {
-
-    let movies = [
-        {
-            imageUrl: 'https://images.unsplash.com/photo-1611419010196-a360856fc42f',
-            title: 'Scary Fog',
-            year: '2003',
-            genres: 'Horror',
-            id: 1
-        }
-    ]
+export default function MoviesList ({movies, onSelect}) {
```

Comment:
Please use the `const MoviesList ` notation with default export at the end of file.